packages/webpack-cli/lib/groups/ConfigGroup.js

```diff
@@ -167,22 +170,19 @@ class ConfigGroup extends GroupHelper {
         if (Object.keys(this.args).some((arg) => arg === 'merge')) {
             const { merge } = this.args;
 
-            const newConfigPath = this.resolveFilePath(merge, 'webpack.base.js');
-            if (newConfigPath) {
-                const configFiles = getConfigInfoFromFileName(newConfigPath);
-                if (!configFiles.length) {
-                    this.opts.processingMessageBuffer.push({
-                        lvl: 'warn',
-                        msg: 'Could not find file to merge configuration with...',
-                    });
-                    return;
-                }
-                const foundConfig = configFiles[0];
-                const resolvedConfig = this.requireConfig(foundConfig);
-                const newConfigurationsObject = await this.finalize(resolvedConfig);
-                const webpackMerge = require('webpack-merge');
-                this.opts['options'] = webpackMerge(this.opts['options'], newConfigurationsObject.options);
+            // try resolving merge config
+            const newConfigPath = this.resolveFilePath(merge);
+
+            if (!newConfigPath) {
+                logger.error("The supplied merge config doesn't exist.");
```

Comment:
We don't need logging error in this case, because it is duplicate, you already see the error in console